roles/openshift_logging/handlers/main.yml

```diff
@@ -3,3 +3,14 @@
   systemd: name={{ openshift.common.service_type }}-master state=restarted
   when: (openshift.master.ha is not defined or not openshift.master.ha | bool) and (not (master_service_status_changed | default(false) | bool))
   notify: Verify API Server
+
+- name: Verify API Server
+  uri:
+    url: "{{ openshift.master.api_url }}/healthz/ready"
+    return_content: yes
+    status_code: 200
+  register: api_available_output
+  until: "'ok' in api_available_output.content"
```

Comment:
@detiber it might be that the end point is available but for some reason the api cluster isn't ready yet, not sure if it can return a different value.... but i'll make the update since it should be fairly safe